packages/server/readme.md

```diff
@@ -40,7 +34,6 @@ To play against the server from the browser client, the server and the browser n
 - Point to the same contract addresses on Ganache. For now, run something like `cp ../wallet/build/contracts/* build/contracts/` after deploying the contracts from the wallet.
 - The server relies on transpiled wallet code. If there are transpile errors when building the server, try transpiling the wallet package by running `npx tsc` in `packages/wallet`.
   Another option is to leave `npx tsc --watch` running in `packages/wallet` to ensure that the transpiled project is always up to date.
-- Start the firebase relay: `yarn firebase-relay:start:dev`
```

Comment:
The readme probably should be updated to contain instructions on how to run the server.

Reply:
I think that's mentioned about with the `yarn server:watch (will rebuild app on file change)`?

Reply:
Yep you are right!